src/transformStateWithClones.js

```diff
@@ -7,7 +7,28 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const newState = { ...state };
+  const res = [];
```

Comment:
res is totally bad name, be more specific with variable names, something like stateHistory would fit perfectly here